client/packages/system/src/ContactTrace/DetailView/DetailView.tsx

```diff
@@ -125,27 +125,29 @@ export const DetailView: FC<DetailViewProps> = ({
 
   useEffect(() => {
     if (contactData) {
-      setSuffix(
-        <span key="patient-contact-trace">
-          <Breadcrumb
-            to={RouteBuilder.create(AppRoute.Dispensary)
-              .addPart(AppRoute.Patients)
-              .addPart(contactData.patient.id)
-              .addQuery({ tab: PatientTabValue.ContactTracing })
-              .build()}
-          >
-            {getLocalisedFullName(
-              contactData.patient?.firstName,
-              contactData.patient?.lastName
-            )}
-          </Breadcrumb>
-          <span>{` / ${contactData.programName} - ${dateFormat.localisedDate(
-            contactData.documentData.datetime
-          )}`}</span>
-        </span>
-      );
+      setBreadcrumbRenderers({
```

Comment:
This pattern to render an additional breadcrumb part seems not great, but a little out of scope?

Reply:
I agree it might be worth rolling this in, doesn't go nicely with my url part lookup idea though...